_schemas/1.0/tests/response/invalid/relationships/to_many_linkage_twice.json

```diff
@@ -0,0 +1,23 @@
+{
+  "data": {
+    "type": "article",
+    "id": "1",
+    "attributes": {
+      "title": "Rails is Omakase"
+    },
+    "relationships": {
+      "author": {
+        "data": [
+          {
+            "type": "people",
+            "id": "9"
+          },
+          {
+            "type": "people",
+            "id": "9"
+          }
+        ]
+      }
```

Comment:
I'm not sure if this is actually invalid. See #1361.

@dgeb We may need to clarify if parallel relationships are supported or not.

Reply:
I agree with you : it's not clear if parallel relationships are supported or not.
When in doubt, I preferred to be as restrictive as possible.
As soon as the situation is clarified, I will change the schema and the test files.

Reply:
As I read that issue and the spec, I think this specific example is indeed invalid; the linked issue relating to a potential `arity` member or other way to uniquely identify them would clear that up.

Reply:
> We are hashing out a final decision re: parallel relationships in [#1361 (comment)](https://github.com/json-api/json-api/issues/1361#issuecomment-1287890619)

It seems that we came to conclusion that parallel relationships are not forbidden. We consider changing that as a breaking change. At least one major implementation relies on it. As we have a strong guarantee to not add any breaking changes, we do not plan to change it.

We plan to add a recommendation that they SHOULD NOT be used. I will create a PR updating the spec as soon as possible. But as it's only a recommendation this should not affect the schema.

Would be great if you could update the schema accordingly.

Reply:
Ok, i will do it next week.

Reply:
@jelhan,  i've updated both schema and tests.